src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -497,20 +587,128 @@ private static List<File> getSubFiles(File file) {
 		return files;
 	}
 
-	private static @Nullable File getScriptFromArgs(CommandSender sender, String[] args) {
-		return getScriptFromArgs(sender, args, Skript.getInstance().getScriptsFolder());
+	private static File @Nullable [] getScriptsFromArgs(CommandSender sender, String[] args) {
+		return getScriptsFromArgs(sender, args, Skript.getInstance().getScriptsFolder());
+	}
+
+	private static File @Nullable [] getScriptsFromArgs(CommandSender sender, String[] args, File directoryFile) {
+		boolean enable = args[0].equals("enable");
+		Map<String, File> scripts = new HashMap<>();
+		for (String currentScript : parseAsCommaSeparatedList(false, true, args)) {
```

Comment:
currentScript and thisScript are very confusing names and could be improved. For example, currentScript isn't a script, it's a script name from the args. The name should reflect that. Likewise, what does this mean in thisScript?